# Service to get the forward kinematics of the robot

float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
float64 j6

---

float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz

int64 ret
string msg
